### PR TITLE
Create page templates

### PR DESCRIPTION
**Pull request by SourabhChakraborty**
Created templates for employer home page, site landing page, and talent portal generic page (for the about page). Future pull requests will add formatting and relevant fields to these page types

## Summary by CodeRabbit

- **New Features**
  - Introduced a new "Talent Portal Landing Page" template for enhanced user experience.
  - Implemented a "Questionnaire" context to facilitate user interaction.

- **Enhancements**
  - Improved the display of top announcements by ordering them based on menu preferences.
  - Enhanced image loading performance with image preloading techniques.

- **Bug Fixes**
  - Adjusted HTML structure and conditional rendering in templates to ensure consistent layout and content display.

- **Documentation**
  - Added comments to clarify the purpose and usage of new templates and contexts.

### CODE REVIEW

**Review by Kinjalpateloti — APPROVED** [review]

File: wp-content/themes/workingnyc/template-employer-home-page.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+/**
+ * Render the view
+ *
+ * @author NYC Opportunity
+ */
+
+$compiled = new WorkingNYC\CompileImgPreload('employer-home-page.twig', $context);
+
+echo $compiled->html;
```
[the remark this answers is not in the document]
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/template-employer-home-page.php:82]: Ignore

File: wp-content/themes/workingnyc/views/employer-home-page.twig [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            <h1 id="page-heading" class="m-0" style="line-height: 0">
+              <svg class="fill-text stroke-text text-brand" width="256" height="79" role="img">
+                <title>Job Ready by Jobs N Y C</title>
+
+                <use href="#logo-jnyc-horizontal-w-ribbon"></use>
+              </svg>
+            </h1>
+          </div>
+
+          {% set featured_image = '<img class="bg-image-square mb-2 tablet:hidden animated fadeIn hidden" loading="eager" style="object-position: ' ~ post.thumbnail.focal_point ~ ';" src="' ~ post.thumbnail.src('tablet') ~ '" alt="' ~ post.thumbnail.alt ~ '"/>' %}
```
[the remark this answers is not in the document]
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/views/employer-home-page.twig:26]: Ignore

File: wp-content/themes/workingnyc/views/employer-home-page.twig [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+          {# The post content can include html tags and CSS classes. The following line is an example of some classes that were used in the past #}
+          {# class="h4 text-center tablet:text-start tablet:text-h3 font-primary text-alt" #}
+          <h1 class="my-3">{{ post.post_content }}</h1>
```
[the remark this answers is not in the document]
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/views/employer-home-page.twig]: Done

File: wp-content/themes/workingnyc/template-talent-portal-landing-page.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Template Name: Talent Portal Home Page
+ *
+ * @author NYC Opportunity
+ */
```
[the remark this answers is not in the document]
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/template-talent-portal-landing-page.php]: Done

File: wp-content/themes/workingnyc/template-talent-portal-landing-page.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * Context
+ *
+ * @author NYC Opportunity
+ */
+
+$context = Timber::get_context();
+
+// TODO: add this when we launch the talent portal
+// $post = Timber::get_post(get_option('page_on_front'));
```
[the remark this answers is not in the document]
Comment (SourabhChakraborty) [reply wp-content/themes/workingnyc/template-talent-portal-landing-page.php:16]: Ignore

**Pull request merged**